catalogs/catalog-model/src/main/java/org/apache/gravitino/catalog/model/ModelCatalogOperations.java

```diff
@@ -96,9 +99,10 @@ public NameIdentifier[] listModels(Namespace namespace) throws NoSuchSchemaExcep
           .toArray(NameIdentifier[]::new);
 
     } catch (NoSuchEntityException e) {
-      throw new NoSuchSchemaException(e, "Schema %s does not exist", namespace);
+      throw new NoSuchSchemaException(e, ErrorMessages.SCHEMA_DOES_NOT_EXIST, namespace);
     } catch (IOException ioe) {
-      throw new RuntimeException("Failed to list models under namespace " + namespace, ioe);
+      throw new RuntimeException(
+          String.format(ErrorMessages.FAILED_TO_LIST_MODELS, namespace), ioe);
```

Comment:
Things like this to refactor the error message is unrelated to this PR and not so worthy to change, please remove all such changes.